Source/WebCore/platform/mediastream/RealtimeMediaSourceSettings.h

```diff
@@ -35,21 +35,21 @@
 
 namespace WebCore {
 
+enum class VideoFacingMode : uint8_t { Unknown, User, Environment, Left, Right };
+enum class DisplaySurfaceType : uint8_t {
```

Comment:
Preexisting but we should probably have a consistent style between the two enums (either one line or one enum per line).